modules/pam_shells/pam_shells.8.xml

```diff
@@ -74,6 +74,14 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>PAM_USER_UNKNOWN</term>
+        <listitem>
+          <para>
+            The does not exist or the user's login shell could not be determined.
```

Comment:
Did you mean "The _user_ does not exist"?

Reply:
you are right